storybook/src/Storybook/FlickableImage.qml

```diff
@@ -0,0 +1,32 @@
+import QtQuick 2.14
+import QtQuick.Controls 2.14
+
+Item {
+    property alias source: image.source
+
+    Rectangle {
+        anchors.fill: parent
+        color: "lightgray"
+    }
+
+    BusyIndicator {
+        anchors.centerIn: parent
+        visible: image.status !== Image.Ready
```

Comment:
Will this stop the animation?
Probably better to use running:
`running: image.status !== Image.Ready`
As per documentation when running is false, the visible property is also set to false.

Reply:
right, changed to use `running`